cloudtrail-check.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# 
+# This script is here to automate (as much as possible) the tasks in
+# https://cloud.gov/docs/ops/maintenance-list/#review-aws-cloudtrail-events
+# 
+# authorized events will include:
+#   a user name like i-17deadbeef1234567
+#   a source IP address within AWS.
+#   an AWS access key starting with ASIA
+# 
+# XXX The source IP range is kind of janky.  It would be better if this
```

Comment:
How janky? What specifically is wrong with it? Can we do handle addresses better?

Reply:
It is janky because it's not handling CIDR notation for realz.  So if you have a 10.10.0.0/17, this is a PITA because you'd have to enumerate 10.10.{0-127}.0 for matching.  It is possible that this could be done with some regex/jq gymnastics, but I couldn't think of a great approach, so I just punted and went with classful matching.